### PR TITLE
Server 0.3

### PR DESCRIPTION
**Pull request by nnance**
Doc updates for Apollo-Server 0.3

### CODE REVIEW

File: source/apollo-server/graphiql.md [hunk 1]
```diff
@@ -50,17 +50,21 @@ app.use('/graphiql', graphiqlConnect({
...
 
-<h2 id="graphiqlHAPI">Using with HAPI</h2>
+<h2 id="graphiqlHapi">Using with Hapi</h2>
 
-If you are using HAPI, GraphiQL can be configured as follows:
+If you are using Hapi, GraphiQL can be configured as follows:
 
 ```js
-import { GraphiQLHAPI } from 'apollo-server';
+import { GraphiQLHapi } from 'apollo-server';
```
Comment (helfer) [inline source/apollo-server/graphiql.md]: isn't it `graphiqlHapi`?

File: source/apollo-server/graphiql.md [hunk 2]
```diff
@@ -50,17 +50,21 @@ app.use('/graphiql', graphiqlConnect({
...
 
 server.register({
-    register: new GraphiQLHAPI(),
-    options: { endpointURL: '/graphql' },
-    routes: { prefix: '/graphiql' },
+  register: GraphiQLHapi,
```
Comment (helfer) [inline source/apollo-server/graphiql.md]: same here, I thought it was `graphiqlHapi`?

File: source/apollo-server/migration-hapi.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    register: new ApolloHAPI(),
+    options: { schema: myGraphQLSchema },
+    routes: { prefix: '/graphql' },
+});
+```
+
+... should be written as follows for Hapi Apollo 0.3.x
+
+```js
+import { ApolloHapi } from 'apollo-server';
```
Comment (helfer) [inline source/apollo-server/migration-hapi.md]: `apolloHapi`?

File: source/apollo-server/migration-hapi.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+The most notable changes are:
+
+- the plugin class has been replaced as a function to be more idiomatic
+- the plugin name has been renamed to use camelcase
+- the options object has been extended to support additional routing options
+
+The following code snippet for Hapi Apollo 0.2.x
+
+```js
+import { ApolloHAPI } from 'apollo-server';
```
Comment (helfer) [inline source/apollo-server/migration-hapi.md]: Should it be `apolloHapi`?

File: source/apollo-server/setup.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ order: 202
 description: How to set up Apollo Server
 ---
 
-Apollo Server exports the `apolloExpress`, `apolloConnect`, `ApolloHAPI` and `apolloKoa` functions which can be used to add a GraphQL HTTP endpoint to your Express, Connect, HAPI or Koa 2 server.
+Apollo Server exports the `apolloExpress`, `apolloConnect`, `ApolloHapi` and `apolloKoa` functions which can be used to add a GraphQL HTTP endpoint to your Express, Connect, Hapi or Koa 2 server.
```
Comment (helfer) [inline source/apollo-server/setup.md]: `apolloHapi`

File: source/apollo-server/setup.md [hunk 6]
```diff
@@ -82,15 +85,15 @@ app.listen(PORT);
...
+<h2 id="apolloHapi">Using with Hapi</h2>
 
-The following code snippet shows how to use Apollo Server with HAPI:
+The following code snippet shows how to use Apollo Server with Hapi:
 
 ```js
-import Hapi from 'hapi';
-import { ApolloHAPI } from 'apollo-server';
+import hapi from 'hapi';
+import { ApolloHapi } from 'apollo-server';
```
Comment (helfer) [inline source/apollo-server/setup.md]: `apolloHapi`

File: source/apollo-server/migration-hapi.md (hunk 4 above)
Comment (nnance) [inline source/apollo-server/migration-hapi.md]: actually this is showing the old version so this is correct

Comment (helfer) [inline source/apollo-server/migration-hapi.md]: Ah, makes sense! My bad.

**Pull request merged**